### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Lharbeeb**

### CODE REVIEW

File: frontend/pages/index.html [hunk 1]
```diff
@@ -1,17 +1,217 @@
...
+      </nav>
+      <div class="socials">
+        <a href="/"><i class="fa-brands fa-facebook-square"></i></a>
+        <a href="/"><i class="fa-brands fa-twitter"></i></a>
+        <a href="/"><i class="fa-brands fa-pinterest"></i></a>
+        <a href="/"><i class="fa-brands fa-instagram"></i></a>
+      </div>
+    </footer>
+  </body>
+</html>
```
Comment (spykelionel) [inline frontend/pages/index.html]: follow class naming rules.
e.g landing page = _lp__
So all your css classes must be prefixed with _lp__
@Lharbeeb

**Review by spykelionel — CHANGES_REQUESTED** [review]
Follow class naming rules

File: frontend/pages/index.html (hunk 1 above)
Comment (Lharbeeb) [reply frontend/pages/index.html]: i thinked we resolved this yesterday, every page will have their unique ccs file so theres no case of having class name conflicts

Comment (spykelionel) [reply frontend/pages/index.html]: No. The backend will merge all files pls. Just give unique names to your class as on the readme.md file pls.

**Pull request merged**